lib/activerecord-multi-tenant/model_extensions.rb

```diff
@@ -71,6 +80,12 @@ def inherited(subclass)
           end
         }
 
+        # Below block adds the following methods to the model that calls it.
+        # partition_key= - returns the partition key for the model.Above in the class getter was defined.
```

Comment:
```suggestion
        # partition_key= - returns the partition key for the model.
```
Not sure what you were trying to get at with that last bit

Reply:
oh, if this referencing something like `model.Above` then use backticks to not make it look like weird punctuation

Reply:
Added some more details